proxy-manager/DOCS.md

```diff
@@ -32,6 +32,7 @@ comparison to installing any other Home Assistant add-on.
 1. Click the "OPEN WEB UI" button and login using:
    `admin@example.com` / `changeme`
 1. Forward port `80` and `443` from your router to your Home Assistant machine.
+1. Update your home assistant configuration's http settings to [`use-x-forwarded-for` and `trusted_proxies`](https://www.home-assistant.io/integrations/http#use_x_forwarded_for)
```

Comment:
- Home Assistant is misspelled.
- `HTTP` is an abbreviation and should be capitalized.
- Keep lines within the 80-char length limit.

This step looks odd and out of place. The problem with it are:

- This is not Home Assistant.
- It assumes the reverse proxy is used with Home Assistant, which doesn't have to be the case. Additionally, we don't provide instructions for other software either.
- We don't provide instructions on how to configure the NGINX Proxy Manager itself either. Why would we provide instructions on how to configure Home Assistant?

Reply:
I'm not sure I follow on why this is irrelevant here.  The first 3 steps are all HA steps, and this one is too.  

The step right above (forward ports 80&443 to your HA machine) clearly assume it is being used for a reverse proxy in front of HA.

How about  a note under install "If using as a reverse proxy for Home Assistant you will need to ...."

Reply:
> The first 3 steps are all HA steps, and this one is too.

The first three steps are add-on installation steps.

> The step right above (forward ports 80&443 to your HA machine) clearly assume it is being used for a reverse proxy in front of HA.

Incorrect.

> How about a note under install "If using as a reverse proxy for Home Assistant you will need to ...."

Maybe as a separate chapter, but it isn't directly related to the add-on installation instructions.

Reply:
I will reformat as chapter (is that a section on doc == installation?

Reply:
i just removed that section, leaving note in torubleshooting.